### DISCUSSION TITLE
[iOS] Bounding Box API

### DISCUSSION THREAD

**Discussion by h0shy** [maplibre/maplibre-native#1098] [Q&A] [answered]
In iOS, the only way to limit the bounding box is via the `shouldChangeFrom` delegate callback. With that, we did not find a way to confine scrolling with a map area which works well on every zoom level. In the Android SDK, there is the option to limit scrolling via `setLatLngBoundsForCameraTarget`. Is there any chance something like this could be implemented for iOS?

**Comment by louwers** [comment, marked as answer]
@h0shy Sure! There are not many iOS developers actively working on new features at the moment. PR welcome though... And feel free to create an issue, maybe someone will pick it up.
